Index.html

```diff
@@ -186,5 +186,253 @@ <h2 class="hero-text-title">I'm a Full Stack Web Developer</h2>
+          </svg>
+
+          <svg class="about-vector25-4">
+            <use xlink:href="images/sprite.svg#icon-Vector25"></use>
+          </svg>
+
+          <svg class="about-vector25-5">
+            <use xlink:href="images/sprite.svg#icon-Vector25"></use>
+          </svg>
+
+          <svg class="about-vector25-6">
+            <use xlink:href="images/sprite.svg#icon-Vector25"></use>
+          </svg>
+
+          <svg class="about-vector26">
+            <use xlink:href="images/sprite.svg#icon-Vector26"></use>
+          </svg>
+
+          <svg class="about-vector28">
+            <use xlink:href="images/sprite.svg#icon-Vector28"></use>
+          </svg>
+
+          <div class="about-hiding-line">
+
+          </div>
+
+          <svg class="about-vector29">
+            <use xlink:href="images/sprite.svg#icon-Vector29"></use>
+          </svg>
+
+          <svg class="about-vector30">
+            <use xlink:href="images/sprite.svg#icon-Vector30"></use>
+          </svg>
+
+          <svg class="about-vector31">
+            <use xlink:href="images/sprite.svg#icon-Vector31"></use>
+          </svg>
+
+          <svg class="about-vector32">
+            <use xlink:href="images/sprite.svg#icon-Vector32"></use>
+          </svg>
+
+          <svg class="about-vector33">
+            <use xlink:href="images/sprite.svg#icon-Vector33"></use>
+          </svg>
+
+          <svg class="about-vector34">
+            <use xlink:href="images/sprite.svg#icon-Vector34"></use>
+          </svg>
+
+          <svg class="about-vector35">
+            <use xlink:href="images/sprite.svg#icon-Vector35"></use>
+          </svg>
+
+          <svg class="about-vector36">
+            <use xlink:href="images/sprite.svg#icon-Vector36"></use>
+          </svg>
+
+          <svg class="about-vector37">
+            <use xlink:href="images/sprite.svg#icon-Vector37"></use>
+          </svg>
+
+          <svg class="about-vector38">
+            <use xlink:href="images/sprite.svg#icon-Vector38"></use>
+          </svg>
+
+          <svg class="about-vector46">
+            <use xlink:href="images/sprite.svg#icon-Vector46"></use>
+          </svg>
+
+          <svg class="about-vector47">
+            <use xlink:href="images/sprite.svg#icon-Vector47"></use>
+          </svg>
+
+          <svg class="about-vector48">
+            <use xlink:href="images/sprite.svg#icon-Vector48"></use>
+          </svg>
+        </div>
+      </div>
+
+      <hr class="hr-about-me">
+
+      <div class="about-me-skills">
+
+        <ul class="about-me-ul">
+          <li id="about-me-skills-language-icon" class="about-me-skills-title-icon"></li>
+          <li class="about-me-skills-title">Languages</li>
+          <li class="about-me-li">Javascript</li>
+          <li class="about-me-li">Ruby</li>
+          <li class="about-me-li">Bash</li>
+          <li class="about-me-li">HTML</li>
+          <li class="about-me-li">CSS</li>
+        </ul>
+
+        <ul class="about-me-ul">
```

Comment:
- [ ] Please make sure the skills section is aligned in the center and matches with the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/166123011-87c99706-84d6-4b59-aa11-3400af88b37f.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/166123019-9cf0cfd6-126f-48fa-8824-b0d6ebc608ea.png)